tests/fullstack-test2/ddl/alter_pk.test

```diff
@@ -62,7 +62,7 @@ mysql> alter table test.t drop primary key;
 mysql> drop table if exists test.t_case;
 ## create table with `source` is nullable
 ## insert some data and left `source` to be empty
-mysql> create table test.t_case (`case_no` varchar(32) not null,`source` varchar(20) default null,`p` varchar(12) DEFAULT NULL,primary key (`case_no`));
+mysql> create table test.t_case (`case_no` varchar(32) not null,`source` varchar(20) default null,`p` varchar(12) DEFAULT NULL,primary key (`case_no`) NONCLUSTERED);
```

Comment:
Introduced by https://github.com/pingcap/tidb/pull/37384 and https://github.com/pingcap/tidb/pull/38447